Marlin/Marlin_main.cpp

```diff
@@ -4504,6 +4519,110 @@ inline void gcode_M503() {
 
 #endif // DUAL_X_CARRIAGE
 
+ // EASY_UI
+#ifdef EASY_UI
+inline void gcode_M700() {
+
+        SERIAL_ECHOLN(" --LEVEL PLATE SCRIPT--");    
+        set_ChangeScreen(true);
+        
+        while(!lcd_clicked()){
+        set_pageShowInfo(0);      
+        lcd_update();        
+        }
+        
+        set_pageShowInfo(1);
+        set_ChangeScreen(true);        
+
+        gcode_G28();
+
+        // prob 1
+        
+        do_blocking_move_to(current_position[X_AXIS], current_position[Y_AXIS],Z_MIN_POS+10);
```

Comment:
Hardcoded 10 is not a good way. Some users have a smaller platform so this could make problems.